README.md

```diff
@@ -2,20 +2,24 @@
 
 This repository contains all Auth0 documentation content (including Quickstarts).
 
-* If you are looking for the application that *hosts* the Docs content, see [auth0-docs](https://github.com/auth0/auth0-docs)
-* If you would like to modify the Management API v2 API docs, see [api2](https://github.com/auth0/api2)
+**Please review the [Contributing Guidelines](CONTRIBUTING.md) before sending a PR or opening an issue.**
 
-Both repositories require team access.
+* If you are looking for the application that *hosts* the Docs content, see [auth0-docs](https://github.com/auth0/auth0-docs).
+* If you would like to modify the Management API v2 API docs, they are generated from the [api2](https://github.com/auth0/api2) repository.
 
-**Please review the [Contributing Guidelines](CONTRIBUTING.md) before sending a PR or opening an issue.**
+Both of the above repositories require team access.
+
+## Editing Docs Content
 
-## Running the Docs Site
+* You can edit Docs content by simply using the GitHub web editor and editing a file. This is best suited for typos and small changes.
+* You can also pull down the `/docs` repo to your computer via Git and edit files in your local editor, before pushing a new branch (or a branch to your own fork of the project). You can then easily come to GitHub.com and start a PR. We will be able to review the changes in a Heroku test application prior to merging.
```

Comment:
Remove "easily"? Just remembering from the WTD conference I went to that many users find "simply" and "easily" to be condescending because it may not be easy for them.